.mailmap

```diff
@@ -477,6 +477,7 @@ Harshil Goel <harshil158@gmail.com>
 Harshil Goel <harshil158@gmail.com> <darkcoderrises@users.noreply.github.com>
 Harshil Meena <harshil.7535@gmail.com>
 Harshit Yadav <harshityadav2k@gmail.com> Harshit Yadav <45384915+hyadav2k@users.noreply.github.com>
+harshrishimiglani <miglaniharshrishi@gmail.com>
```

Comment:
How do you want your name and email address to be recorded in the AUTHORS file?

If your name is "Harshrishi Miglani" then you can do it like this:
```
Harshrishi Miglani <miglaniharshrishi@gmail.com> harshrishimiglani <miglaniharshrishi@gmail.com>
Harshrishi Miglani <miglaniharshrishi@gmail.com> harshrishimiglani <97667942+harshrishimiglani@users.noreply.github.com>
```
That specifies what should be recorded in the AUTHORS file (Harshrishi Miglani) and what metadata identifies the commits associated with that author. The noreply email address is found in commits made through the GitHub web UI.

Reply:
Hi @oscarbenjamin I would like my name to be written as "Harsh Rishi Miglani" and email to be "miglaniharshrishi@gmail.com".

Reply:
After adding these lines you need to run `bin/mailmap_check.py` which will reorder the lines in the file.

Reply:
Alright I will try to accommodate the suggested changes.